src/idgen.c

```diff
@@ -353,6 +354,9 @@ Msgtable msgtable[] =
     { "basic_ostream" },
     { "basic_iostream" },
     { "char_traits" },
+
+    // Compiler recognized UDA's
+    { "udaSelector", "core.attribute.selector" },
```

Comment:
Would it be better not to detect the special UDA by name, but have the user import e.g. `core.objc`, which contains a symbol `selector`, and let the compiler detect this special symbol? Then it wouldn't make existing `@selector` UDAs invalid, at the cost of needing an explicit import.